ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/leaf-spine-topology.md

```diff
@@ -149,6 +149,9 @@ l3leaf:
       # Spanning tree priority.
       spanning_tree_priority: < spanning-tree priority -> default 32768 >
 
+      # Spanning tree priority.
+      spanning_tree_root_super: < true | false -> default false >
```

Comment:
you template is not producing "root_super: false" in case this value is unset. I think we should remove the "-> default false" from documentation.